models.py

```diff
@@ -420,50 +305,29 @@ def checkout(self, order_id, order_status, order_address, checkout_time, estimat
                             CustOrderStatus.estimated_time: estimated_time,
                             CustOrderStatus.bill_amount: bill_amount}, 
                             synchronize_session=False)
-        try:
-            session.commit()
-            return checkout_update
-        except:
-            session.rollback()
-            raise Exception("Checkout not completed!")
-        finally:
-            session.expunge_all()
-            session.close()
-
-
-    def cancel_order(self, order_id, order_status, session):
+        session.commit()
+        return checkout_update
+    
+    def cancel_order(self, session, order_id, order_status):
         """ Cancel order """
 
         update = session.query(CustOrderStatus).\
                     filter(CustOrderStatus.order_id == order_id).\
                     update({CustOrderStatus.order_status: order_status}, 
                     synchronize_session=False)
-        try:
-            session.commit()
-            return update
-        except:
-            session.rollback()
-            raise Exception("Checkout not completed!")
-        finally:
-            session.expunge_all()
-            session.close()
-
-
-    def view_orders_status(self, order_id, session):
+        session.commit()
+        return update
+    
+    def view_orders_status(self, session, order_id):
```

Comment:
This function is repeated twice for employee and customer. It's code repetition! It's not good. So you can actually create a function called `view_order_status()` which is not part of any class and then call it in the Employee and Customer class.

Reply:
created a class for the common functions that the employee and customer can use.